src/heads-up/components/property-data.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <span>{{ tooltipMessage }}</span>
+</template>
+<script>
+
+function isDefined(tag) {
+  const isImg = tag === 'og:image'
+  return `There is no ${tag} defined. You can create the ${tag} in the <head> like <meta property="${tag}" content="${isImg?'your source':'your content'}."> `
+}
+
+function isTooLong(length) {
+  return `The content has more than ${length} characters. Consider shorten your content. `
+}
+
+function isRequired(tag) {
+  return `The ${tag} is required to create an unfurling link on the platform. `
+}
+
+function isBigImg(tag, smallVariant, imgSize, requiredSize) {
+  if (smallVariant) {
+    return`The ${tag} is ${imgSize.width} by ${imgSize.height}px, where ${requiredSize.width} by ${requiredSize.height}px is required for a big unfurling preview. `
```

Comment:
Probably change this: `${imgSize.width} by ${imgSize.height}px` to `${imgSize.width}px by ${imgSize.height}px` to be completely sure.